pychunkedgraph/graph/operation.py

```diff
@@ -453,17 +450,21 @@ def execute(
                     timestamp=None,
                     operation_ts=override_ts if override_ts else timestamp,
                     status=attributes.OperationLogs.StatusCodes.EXCEPTION.value,
-                    exception=repr(format_exception(*exc_info())),
+                    exception=repr(err),
                 )
                 self.cg.client.write([log_record_error])
                 raise Exception(err)
-            return self._write(
-                root_lock,
-                override_ts if override_ts else timestamp,
-                new_root_ids,
-                new_lvl2_ids,
-                affected_records,
-            )
+
+            with TimeIt("persisting changes"):
+                result = self._write(
+                    root_lock,
+                    override_ts if override_ts else timestamp,
+                    new_root_ids,
+                    new_lvl2_ids,
+                    affected_records,
+                )
+                print("new root ids", result.new_root_ids)
```

Comment:
Maybe this print statement is useful. Just leaving this note here in case it was left in by accident

Reply:
It was left on purpose so that it shows up in edit logs along with timings, used to be in `edits.py` but moved it here to preserve indentation within `TimeIt` block.